Mathlib/RingTheory/DedekindDomain/Ideal/Lemmas.lean

```diff
@@ -1002,6 +1023,8 @@ theorem primesOver_finite : (primesOver p B).Finite := by
   · rw [← coe_primesOverFinset hpb B]
     exact (primesOverFinset p B).finite_toSet
 
+noncomputable instance : Fintype (p.primesOver B) := Set.Finite.fintype (primesOver_finite p B)
```

Comment:
Do we really need `Fintype` instead of `Finite`?

Reply:
Well, I want the formula 
```lean
map (algebraMap S R) p = ∏ P ∈ p.primesOver R, P ^ p.ramificationIdx (algebraMap S R) P 
``` 
to typecheck on line 800 of `RingTheory.DedekindDomain.Factorization` so I thought having a `Fintype` was necessary. Is there another way to proceed?